client/verta/verta/data_types/_numeric_value.py

```diff
@@ -54,3 +54,11 @@ def _from_dict_inner(cls, d):
             value=data["value"],
             unit=data["unit"],
         )
+
+    def dist(self, other):
+        if not isinstance(other, type(self)):
+            raise TypeError(
+                "`other` must be type {}, not {}".format(type(self), type(other))
+            )
+
+        return self._value - other._value
```

Comment:
I'd like to suggest we use absolute percentage change here instead of raw difference.

- This difference isn't a distance function, which will be fine once we let users define the direction of comparison (above or below threshold), but right now we don't offer that and so I would assume +20% change and -20% change should be equally alert worthy.
- It's harder to set a threshold for a raw difference than for a percentage change unless you're intimately familiar with your data.

Reply:
For a reference alerter, it makes sense to me to make this a percentage change (and then alert if that absolute percentage change is greater than `threshold`.

I'll need to figure out what to do for the fixed alerter, since the original plan was to use `NumericValue.dist()` for that, but that threshold comparison should be a raw difference